cmd/tsbs_generate_queries/databases/clickhouse/devops_test.go

```diff
@@ -0,0 +1,86 @@
+package clickhouse
+
+import (
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestDevopsGetHostWhereWithHostnames(t *testing.T) {
+	cases := []struct {
+		desc      string
+		hostnames []string
+		useTags   bool
+		want      string
+	}{
+		{
+			desc:      "single host - no json or tags",
```

Comment:
seems like these test descriptions can be updated to remove mention of `json`

Reply:
Yes, agree, there is no JSON support in ClickHouse atm.
Updated.